### ISSUE TITLE
internal process error (4294930435) + multiple ledger application in task manager

### ISSUE THREAD

**Issue by IAmSold** [LedgerHQ/ledger-live-desktop#3570]
#### Ledger Live Version and Operating System

- tested on Ledger Live 2.21.3
- Platform and version: Windows 10

#### Expected behavior
Before the 2.21.3 update, there was no problem with accessing manager or receiving funds to accounts. When accessing the manager, it will ask for access confirmation with the ledger device and once I do, I would have access to the manager without an issue. When trying to verify the address of an account when I click "receive", it will ask to connect the ledger device to the desktop on step 2. After doing so, it will ask to verify the address shown on the ledger device. 

#### Actual behavior
When I try to access manager, it will either notify me that the page is "Loading..." or it provides me with an internal process error (4294930435). This is also the case when clicking "receive " in accounts. In addition, whenever I close ledger live, 3 ledger live applications are running in my task manager. I have to manually delete them from my task manager in order to open the application again. 

#### Steps to reproduce the behavior

![Screenshot (123)](https://user-images.githubusercontent.com/34350933/107867909-023f7500-6e34-11eb-94b4-ef6fd92ebff1.png)
![Screenshot (120)](https://user-images.githubusercontent.com/34350933/107867911-079cbf80-6e34-11eb-820c-6a5a15b78786.png)

**Comment by gre** [comment]
Hi, thanks a lot for raising this issue. 

could you send your logs to our support team and mention this github issue? https://support.ledgerwallet.com/hc/en-us/requests/new?ticket_form_id=248165  🙏  basically you can press Ctrl+E just after the problem occurs. ( alternatively if you don't want to send the logs , you can see the JSON yourself on https://ledger-live-tools.now.sh/logsviewer )

**Comment by IAmSold** [comment]
yes will do. I found out that the atom app was the reason for the issues. After deleting it, all the issues were resolved. It might have something to do with path derivation but I will notify ledger support about it.
